driver-core/src/main/com/mongodb/annotations/Evolving.java

```diff
@@ -29,11 +29,13 @@
  * This, in turn, means that implementing interfaces or extending classes annotated with {@link Evolving} bears the risk
  * of doing extra work during upgrades.
  * Using such program elements is no different from using ordinary unannotated program elements.
- * Note that the presence of this annotation implies nothing about the quality or performance of the API in question.
+ *
+ * <p>This annotation does not imply that the API is experimental or
+ * {@link Beta}, or that the quality or performance of the API is inferior.
  */
 @Retention(RetentionPolicy.CLASS)
 @Target(ElementType.TYPE)
 @Documented
-@Evolving
+@Sealed
```

Comment:
The approach we have in the team is to try and avoid doing drive-by changes. I suggest creating a separate PR with the `master` branch being its target as these changes do not belong with the work on the expressions API.

P.S. Just for information, in case you decide to propose the changes in a different PR (as this PR is not a place for them), the changes seem quite questionable to me:

1. I don't understand what `@Sealed` means here, and I doubt it is the right thing to do (I left a comment on `Sealed.java` about this).
2. I don't understand removing `@Evolving`. An explanation for it may be a good start.
3. As far as I can see, the change of the wording brings nothing.